app/models/custom/proposal.rb

```diff
@@ -0,0 +1,19 @@
+require_dependency Rails.root.join("app", "models", "proposal").to_s
+
+Proposal.class_eval do
+  scope :by_geozone_id, ->(geozone_id) { where(geozone_id: geozone_id) }
+  scope :sort_by_oldest, -> { reorder(:created_at) }
+
+  class << self
+    alias_method :consul_proposals_orders, :proposals_orders
+
+    def proposals_orders(user)
+      consul_proposals_orders(user) << "oldest"
+    end
+
+    def allowed_filter?(filter, value)
+      return if value.blank?
+      ["official_level", "date_range", "geozone_id"].include?(filter)
```

Comment:
`official_level` doesn't seem to be necessary since we've removed the filter :thinking:.

Reply:
Nice catch! 👌🏼